.env.test.example

```
@@ -55,7 +55,8 @@ UPSTASH_REDIS_REST_URL=
 
 # ---- AI providers / gateway ------------------------------------------------
 AI_GATEWAY_API_KEY=
-AI_GATEWAY_URL=https://ai-gateway.vercel.sh/v1
+AI_GATEWAY_ALLOWED_HOSTS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_GATEWAY_ALLOWED_HOSTS key should go before the AI_GATEWAY_API_KEY key

(UnorderedKey)